dev/cuda/Makefile

```diff
@@ -8,8 +8,20 @@ ifeq ($(NVCC),)
 		$(error nvcc not found.)
 endif
 
+ifneq ($(CI),true) # if not in CI, then use the GPU query
+  ifndef GPU_COMPUTE_CAPABILITY # set to defaults if: make GPU_COMPUTE_CAPABILITY= 
+    GPU_COMPUTE_CAPABILITY = $(shell __nvcc_device_query) # assume if NVCC is present, then this likely is too
```

Comment:
at least on my system, it actually is not present

Reply:
Can remove this - per the below comment too.

Reply:
don't think we need to remove this; I think its the difference between the ubuntu cuda package (which is on PATH in my system) and those from the nvidia sources (which does have this utility in its bin folder; it's just not on PATH in my setup)
I think it would still break on my system because of the two different GPUs, i.e., its return value is  `86,89` 
Having two different GPUs might not be too uncommon in a workstation setup, where you have on cheap(ish) GPU handling the actual display output, and a second GPU for the heavy compute.